### PR TITLE
Add simple done, pending and total counts to main bar

### PR DESCRIPTION
**Pull request by brenodt**
Issue: Adding item count for done, pending and total on the main tab.
 
# Relevant motivation and context
I propose to add these three fields stacked vertically, as follows:
image

The reason is that when they are rendered horizontally they take up the title space, so _Blisser_ disappears
image

# Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
 
# Checklist
- Have you set a PR title that is short and concise?
- Have you used labels like `wip`, `review`, `backend`, `unity`, etc...?
- Have you performed a self-review of your own code?
- Have you commented your code, particularly in hard-to-understand areas?
- Have you made corresponding changes to the documentation?
- Have you added tests that prove your fixes are effective and work as expected?
- Have you ran new and existing tests and they pass locally with my changes?

### CODE REVIEW

File: mobile/blisser/lib/src/views/main_view.dart [hunk 1]
```diff
@@ -46,8 +61,27 @@ class _ItemListState extends State<ItemList> {
             image: AssetImage('lib/assets/images/simple_logo.png'),
           ),
         ),
-        title: _TitleAndButtons(),
+        title: Text(local.translate('main_view', 'title')),
         actions: <Widget>[
+          Column(
```
Comment (diegoseso) [inline mobile/blisser/lib/src/views/main_view.dart:66]: Isolate styling

**Comment by diegoseso** [comment]
remember to recall everything from pending to open

**Comment by diegoseso** [comment]
pending -> open and also remember to change to i18n the naming and isolating the styling

**Comment by brenodt** [comment]
@diegoseso This is the final result:
image

I also fixed the search method to be case-insensitive (as it wasn't before, my bad).
All the fixes we talked about are in place. Let me know if you'd like me to change something else.

**Review by diegoseso — APPROVED** [review]

**Pull request merged**